cmd/serve.go

```diff
@@ -37,6 +37,7 @@ const (
 	s3SecretKeyFlag              = "s3-secret-key" //nolint: gosec
 	s3RegionFlag                 = "s3-region"
 	s3BucketFlag                 = "s3-bucket"
+	s3UsePathStyle               = "s3-use-path-style"
```

Comment:
this is a breaking change. You will need to add the inverse flag. I.e., use `s3-use-subdomain-style`, otherwise any existing user upgrading the version will end up with a broken setup.